src/modules/service-catalog/translations/en-us.yml

```diff
@@ -118,3 +118,21 @@ parts:
       title: "Label indicating number of services currently being listed, the default plural form."
       screenshot: "https://drive.google.com/file/d/1uP3fyDR1vE2Z8lHXiNXaZD7LeTEgLCSw/view?usp=drive_link"
       value: "{{count}} services"
+  - translation:
+      key: "service-catalog.search-services"
+      title: "Search for services in the Service List"
+      screenshot: "https://drive.google.com/file/d/1JHDB_jxDaL0ozmpUTYIhYMaCm4-4SN27/view?usp=drive_link"
+      value: "Search for services"
+      wip: true
```

Comment:
Are we waiting for approval of these strings?
Otherwise we can remove the `wip` tag

Reply:
I'm not sure if the design will change. However, considering that the changes we need are related with the clear all button, I think we can assume the rest of the strings should be final. I'll remove the wip.